docker-stack.yml

```diff
@@ -169,41 +173,49 @@ services:
       replicas: 1
       labels:
         - "traefik.enable=true"
-        - "traefik.http.routers.images.rule=Host(`images.local.wholetale.org`)"
+        - "traefik.http.routers.images.rule=Host(`images.${domain}`)"
         - "traefik.http.routers.images.rule=Method(`GET`)"
         - "traefik.http.routers.images.entrypoints=websecure"
         - "traefik.http.routers.images.tls=true"
         - "traefik.http.services.images.loadbalancer.server.port=5000"
         - "traefik.http.services.images.loadbalancer.passhostheader=true"
         - "traefik.docker.network=wt_traefik-net"
 
-#  celery_worker:
-#    image: wholetale/gwvolman
-#    networks:
-#      - celery
-#    labels:
-#      - "traefik.enable=false"
-#    environment:
-#      - GIRDER_API_URL=https://girder.local.wholetale.org/api/v1
-#      - HOSTDIR=/host
-#      - DOMAIN=local.wholetale.org
-#      - TRAEFIK_NETWORK=wt_traefik-net
-#      - TRAEFIK_ENTRYPOINT=https
-#      - REGISTRY_USER=fido
-#      - REGISTRY_URL=https://registry.local.wholetale.org
-#      - REGISTRY_PASS=secretpass
-#    volumes:
-#      - /var/run/docker.sock:/var/run/docker.sock:ro
-#      - /:/host
-#      - /var/cache/davfs2:/var/cache/davfs2
-#      - /run/mount.davfs:/run/mount.davfs
-#    cap_add:
-#      - SYS_ADMIN
-#      - SYS_PTRACE
-#    devices:
-#      - /dev/fuse:/dev/fuse
-#    command: -Q manager,celery,xa3bl9runqzk8dgbl5gc6nlnq --hostname=xa3bl9runqzk8dgbl5gc6nlnq
-
+  celery_worker:
+    image: wholetale/gwvolman
+    networks:
+      - celery
+    labels:
+      - "traefik.enable=false"
+    environment:
+      - GOSU_USER=wtuser:1000:${docker_group}
+      - DOMAIN=${domain}
+      - DEV=true
+      - REPO2DOCKER_VERSION=${r2d_version}
+      - TRAEFIK_NETWORK=wt_traefik-net
+      - TRAEFIK_ENTRYPOINT=websecure
+      - REGISTRY_USER=${registry_user}
+      - REGISTRY_URL=https://registry.${domain}
+      - REGISTRY_PASS=${registry_pass}
+      - WT_LICENSE_PATH=${PWD}/volumes/licenses
+      - WT_VOLUMES_PATH=${PWD}/volumes
+      - MATLAB_FILE_INSTALLATION_KEY=${matlab_file_installation_key}
+      - SWARM_NODE_ID={{.Node.ID}}
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+      - ./volumes/tmp:/tmp
+      - ./src/gwvolman:/gwvolman
+      - ./src/girderfs:/girderfs
+      - type: bind
+        source: ${PWD}/volumes
+        target: ${PWD}/volumes
+        bind:
+          propagation: rshared
```

Comment:
Why do we need volumes in gwvolman?

Reply:
Took a look at the code and RRuns are writing `.docker_stats`, `.stdout` etc directly to run's workspace.

Reply:
I see it now.